From: Zijun Hu <quic_zijuhu@quicinc.com>
Subject: mm/mmu_gather: remove needless return in void API tlb_remove_page()
Date: Fri, 21 Feb 2025 05:02:06 -0800

Remove needless 'return' in void API tlb_remove_page() since both the API
and tlb_remove_page_size() are void functions.

Link: https://lkml.kernel.org/r/20250221-rmv_return-v1-1-cc8dff275827@quicinc.com
Signed-off-by: Zijun Hu <quic_zijuhu@quicinc.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/asm-generic/tlb.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/asm-generic/tlb.h~mm-mmu_gather-remove-needless-return-in-void-api-tlb_remove_page
+++ a/include/asm-generic/tlb.h
@@ -503,7 +503,7 @@ static inline void tlb_remove_page_size(
 
 static inline void tlb_remove_page(struct mmu_gather *tlb, struct page *page)
 {
-	return tlb_remove_page_size(tlb, page, PAGE_SIZE);
+	tlb_remove_page_size(tlb, page, PAGE_SIZE);
 }
 
 static inline void tlb_remove_ptdesc(struct mmu_gather *tlb, void *pt)
_
